packages/grafana-ui/src/components/Logs/getLogRowStyles.ts

```diff
@@ -136,6 +136,13 @@ export const getLogRowStyles = stylesFactory((theme: GrafanaTheme, logLevel?: Lo
       width: 12.5em;
       padding-right: 1em;
     `,
+    logsRowLabels: css`
+      label: logs-row__labels;
+      display: table-cell;
+      white-space: nowrap;
+      width: 20%;
```

Comment:
@davkal I think that we need to set fixed width (e.g. `10em`/`15em`/`20em`)  for this, as otherwise have dancing layout
![image](https://user-images.githubusercontent.com/30407135/70620566-eb8e0480-1c17-11ea-9805-3cd839fb78db.png)